js/hang-ui/src/Components/stats/handlers/audio.ts

```diff
@@ -0,0 +1,112 @@
+import type { HandlerContext } from "../types";
+import { BaseHandler } from "./base";
+
+/**
+ * Handler for audio stream metrics (channels, bitrate, codec)
+ */
+export class AudioHandler extends BaseHandler {
+	/** Polling interval in milliseconds */
+	private static readonly POLLING_INTERVAL_MS = 250;
+	/** Display context for updating metrics */
+	private context: HandlerContext | undefined;
+	/** Polling interval ID */
+	private updateInterval: number | undefined;
+	/** Bound callback for display updates */
+	private updateDisplay = () => this.updateDisplayData();
+	/** Previous bytes received for bitrate calculation */
+	private previousBytesReceived = 0;
+	/** Previous timestamp for accurate elapsed time calculation */
+	private previousWhen = 0;
+
+	/**
+	 * Initialize audio handler with polling interval
+	 */
+	setup(context: HandlerContext): void {
+		this.context = context;
+		const audio = this.props.audio;
+
+		if (!audio) {
+			context.setDisplayData("N/A");
+			return;
+		}
+
+		this.updateInterval = window.setInterval(this.updateDisplay, AudioHandler.POLLING_INTERVAL_MS);
```

Comment:
we're using the wrapped `updateDisplay` to retain scope, right? We could also do `updateDisplayData.bind(this)`, right?